rqt_py_common/src/rqt_py_common/plugin_container_widget.py

```diff
@@ -75,8 +74,11 @@ def __init__(self, plugin_widget,
         """
         super(PluginContainerWidget, self).__init__()
 
-        ui_file = os.path.join(rospkg.RosPack().get_path('rqt_py_common'),
-                               'resource', 'plugin_container.ui')
+        self._logger = rclpy.logging.get_logger("PluginContainerWidget")
+        _, package_path = get_resource('packages', 'rqt_py_common')
```

Comment:
This could be replaced with a call to qt_gui.ros_package_helper.get_package_path()

Reply:
It could but unless you feel strongly about this I'd rather not change it.